### ISSUE TITLE
Pretrained Big Sleep

### ISSUE THREAD

**Issue by ibayramli2001** [lucidrains/big-sleep#42]
When I run `!dream "something"` on the given colab, it seems to start the training process and displays epochs / losses. Is there a way to simply look at the output of the model using pretrained weights and retrain everything from scratch?

**Comment by htoyryla** [comment]
There's no single pretrained network here to make images from text. There's BigGAN to make images and CLIP to evaluate how an image matches a given text. Therefore one has to iterate to find an image which matches the prompt, that's why you are seeing epochs and losses. The loss measures how well the current image matches the text, and the goal is to minimise it. The models are not being trained at all.

**Comment by illtellyoulater** [comment]
Just a quick related question: how much loss is to be considered good enough, on average with big-sleep? I mean I am seeing that around -36 any further progress becomes really hard... so is that considered a good loss amount? Or are you seeing different results?
